server/src/main/java/com/genymobile/scrcpy/CleanUp.java

```diff
@@ -100,15 +101,27 @@ private void runCleanUp(Options options) {
         boolean powerOffScreen = options.getPowerOffScreenOnClose();
         int displayId = options.getDisplayId();
 
+        int restoreDisplayImePolicy = -1;
+        if (displayId > 0) {
```

Comment:
This is only for secondary displays? (not the main display?)

Reply:
Yes, the main display must be able to display IME normally.